e2e/pages/StageOne/ConsentPage.ts

```diff
@@ -0,0 +1,136 @@
+import { Locator, Page } from "@playwright/test";
+import { assertComponentsVisible } from "../../utils/visibilityUtils";
+import { assertErrorUtil, assertErrorsHidden } from "../../utils/errorUtils";
+
+export default class ConsentPage {
+  readonly page: Page;
+  // text
+  readonly progressDisplay: Locator;
+  readonly consentPageHeading: Locator;
+  readonly consentParagraph1: Locator;
```

Comment:
Try and avoid using numbers in your variable/locator names if possible.
If you can't be more specific then use 'consentParagraphFirst', bePartOfResearchLinkFirst etc instead